RussianFilter/sections/whitelist.txt

```diff
@@ -36,6 +36,8 @@ beko.co.uk#@#.topbanner
 @@||vidalak.com/video.css$domain=multplayer.ru
 @@||vidalak.com/api/vpaid^$domain=multplayer.ru
 @@||vidalak.com/player/vpaid^$domain=multplayer.ru
+! https://github.com/AdguardTeam/AdguardFilters/issues/35955
+@@||ads.businessmixer.net^$domain=ads.businessmixer.net
```

Comment:
Please also add rule for iOS/Safari/content blocker. Site will not open even with that rule.

Reply:
@BlazDT why? It works in Safari/iOS.

Reply:
in any case this site has 0 visitors.

Reply:
Normally iOS blocks it as it contains ads in its url.